test/functional/payment_authorisation_api_test.rb

```diff
@@ -78,4 +78,15 @@ def test_recurring_payment_api_request
     assert response.authorised?
     assert response.psp_reference
   end
+
+  def test_list_recurring_details_api_request
+    response = @client.list_recurring_details(
+      recurring: { contract: "RECURRING" },
+      merchant_account: 'VanBergenORG',
+      shopper_reference: 'willem42',
+    )
+
+    assert response.references.any?
+    assert response.details.any?
```

Comment:
Can we make these assertions a bit more specific? What do you expect the items in `references` and `details` to look like?